src/contact.coffee

```
@@ -79,7 +79,8 @@ Zap.create_contact_pre_write = (bundle, contact_type) ->
     email_address_type ?= "Work"
     data.email_addresses = [{"name": email_address_type, "address": object.email_address.address}]
 
-  if object.address? && object.address.street? && object.address.city?
+  if object.address? && (object.address.street? || object.address.city? || 
```

Good point. This works, if address is empty it won't be added to the request.

But I'm kind of confused:
 
"CoffeeScript's existential operator ? returns true unless a variable is null or undefined" - coffescript docs, but the translated js for `object.address?` is 
``` javascript
 if (object.address != null) { ...
```
I would think this would throw an "Undefined" error.